templates/BlueGrey/compare.tmpl

```diff
@@ -58,4 +58,6 @@
       [websvn-endtest]
     [websvn-endlisting]
   [websvn-endtest]
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
```

Comment:
Not good, maybe not ideal in corporate environment where deployments must be selfcontained.

Reply:
We can download the version and include it in the javascript folder. That is what i have done in my organization.

Reply:
This is prefered.

Reply:
Done.

Reply:
Bundling jQuery the current way is WRONG! Because a revision number or stuff like that is missing, one can't influence anymore if an ever updated version of jQuery will be downloaded by the browser or not. The former implementation referencing the file from some CDN was the better and more standard approach. A lot of web-apps do so and because of that, chances are high that some browser already has the requested version of jQuery in its own cache. This even reduces load on the corporate IT.

@michael-o Which problem do you hope to solve with bundling jQuery? We really should NOT do so.

Reply:
@ams-tschoening Some people dislike serving resources from outside their network. There is no real problem here.

Reply:
Those things can be addressed when having a proper JS-build with package management, cache busting etc. in place. We shouldn't care now unless anyone has a problem, which in my opinion is pretty unlikely. Bundling like currently makes the app only less reliable and means more work for us when jQuery needs to be updated for some reason. Lot's of apps don't use jQuery at all anymore as well.

I strongly suggest using the following line again:

    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>

Reply:
Probably @michael-o is referring to the corporate networks that are isolated. Like I have in our lab. They cannot go outside the firewall and hence will never be able to fetch this js file. These networks are strictly for source code maintenance and testing. Maybe to support those this would makes sense to bundle together. 
But again with more and more corporate outsourcing the source code servers to cloud maybe there is no need to bundle it with this websvn code. 
I am fine with either. But i would prefer this
`<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>`
Let me know which one to do.

Reply:
If the external file works for you, please use the external one. If not, bundle it, but at least renamed to contain the version number, like `jquery.min.1.9.1.js` or something.

Reply:
I have bundled it for now as in my lab we dont have a network connection so the external one is not working. I have bundled it and renamed it to jquery.min.1.9.1.js so that the name version is clear. :-)